apps/src/gamelab/ValidationSetup.interpreted.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+function levelSuccess(testResult) {
+  __validationState = 'succeeded';
+  __validationResult = testResult;
+}
```

Comment:
Can I auto-pass any level with the following code? 😄

![image](https://user-images.githubusercontent.com/413693/39787564-cd618278-52da-11e8-83ef-8667f090a409.png)

Reply:
I should catch you with this line just before the validation code runs: https://github.com/code-dot-org/code-dot-org/blob/41a1ee4b8ba5ae1e97e09a373520b213d799680b/apps/src/gamelab/GameLab.js#L1209

Though if you try to use `__validationState` in your program it'll get reset every tick. The task of adding more reserved keywords becomes ever more pressing.

Reply:
I'm pretty sure we can handle it at the blockly code generation layer by adding "reserved words":
https://github.com/code-dot-org/blockly/blob/00b15c2a78946cdd7d17e808a594c99a6b54bb68/core/code_generation/generator.js#L299-L306

This will then generate slightly different variable names if you try to make a variable name that collides with a reserved word:
https://github.com/code-dot-org/blockly/blob/00b15c2a78946cdd7d17e808a594c99a6b54bb68/core/code_generation/names.js#L110-L131

Reply:
Nice, it looks like we're already doing this!  Maybe add these as reserved words, and remove the double-underscore?

https://github.com/code-dot-org/code-dot-org/blob/bfa51eb8270179d2139531bc85caf98ecc0d6c89/apps/src/gamelab/GameLab.js#L467-L469